### PR TITLE
Update climate_HassClimateSetTemperature.yaml

### PR DESCRIPTION
**Pull request by OliverSkivdal**

### CODE REVIEW

**Comment by synesthesiam** [comment]
@LaStrada

File: sentences/nb/climate_HassClimateSetTemperature.yaml [hunk 1]
```diff
@@ -2,4 +2,7 @@ language: 'nb'
 intents:
   HassClimateSetTemperature:
     data:
-      - sentences: []
\ No newline at end of file
+      - sentences:
+         - "sett temperaturen [i <area>] til <temperature>"
+         - "sett temperaturen til <temperature>"
+         - "(varm opp | kjøl ned) <area> til <temperature>"
```
[the remark this answers is not in the document]
Comment (LaStrada) [reply sentences/nb/climate_HassClimateSetTemperature.yaml:8]: This line still need to be fixed.
```suggestion
         - "(varm opp|kjøl ned) <område> til <temperature>"
```

File: sentences/nb/climate_HassClimateSetTemperature.yaml [hunk 2]
```diff
@@ -2,4 +2,7 @@ language: 'nb'
 intents:
   HassClimateSetTemperature:
     data:
-      - sentences: []
\ No newline at end of file
+      - sentences:
+         - "sett temperaturen [i <area>] til <temperature>"
+         - "sett temperaturen til <temperature>"
```
Comment (LaStrada) [inline sentences/nb/climate_HassClimateSetTemperature.yaml:7]: These lines are not needed anymore.

**Review by LaStrada — CHANGES_REQUESTED** [review]
This PR need to resolve some conflicts and changes requested.

**Comment by LaStrada** [comment]
This PR is very outdated, and there has been major changes since this was opened.

**Pull request closed without merging**